xclim/data/fr.json

```diff
@@ -975,6 +975,12 @@
     "description": "Indice humidex décrivant la température ressentie par une personne en réponse à l'humidité relative.",
     "long_name": "indice humidex"
   },
+  "HEAT_INDEX": {
```

Comment:
@aulemahal I've created this in case you want to make a commit suggestion for the french version of the description. Is there anywhere else where this is needed?

Reply:
Good, I'll find something! That's all for the translations, thanks.